src/main/java/main/resources/PostTweetResource.java

```diff
@@ -0,0 +1,51 @@
+package main.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+
+import javax.ws.rs.*;
```

Comment:
A nitpick, but it's always best to not import everything under the sun when you only need a few packages - if you're using Intellij, you can use `ctrl + option + o` to optimize imports